<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color : black;
font-size : 20px;</string>
      </property>
      <property name="title">
       <string>图源</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="startEndElementBtn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/assets/icon/icon-roundedrectangle.png);
background-repeat: no-repeat;
background-position: center;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="processElementBtn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/assets/icon/icon-rectangular.png);
background-repeat: no-repeat;
background-position: center;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="subprocessElementBtn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/assets/icon/icon-subprocesselement.png);
background-repeat: no-repeat;
background-position: center;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="dataElementBtn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/assets/icon/icon-parallelogram.png);
background-repeat: no-repeat;
background-position: center;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="decisionElementBtn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/assets/icon/icon-diamond.png);
background-repeat: no-repeat;
background-position: center;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="documentElementBtn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/assets/icon/icon-documentelement.png);
background-repeat: no-repeat;
background-position: center;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QPushButton" name="connectorElementBtn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/assets/icon/icon-circle.png);
background-repeat: no-repeat;
background-position: center;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea_Painter">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_main">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>10262</width>
            <height>10262</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="Canvas" name="canvas" native="true">
             <property name="minimumSize">
              <size>
               <width>10240</width>
               <height>10240</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>导出</string>
     </property>
     <addaction name="actionSaveAsSVG"/>
     <addaction name="actionSaveAsPNG"/>
    </widget>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSetBack"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionFillColor"/>
    <addaction name="actionLineColor"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAboutus"/>
    <addaction name="actionReadMe"/>
    <addaction name="actionkaiyuanxieyi"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolNew"/>
   <addaction name="actionSaveTool"/>
   <addaction name="separator"/>
   <addaction name="actionToolCopy"/>
   <addaction name="actionToolCut"/>
   <addaction name="actionToolPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFontFamily"/>
   <addaction name="actionFontBold"/>
   <addaction name="actionFontItalic"/>
   <addaction name="actionFontUnder"/>
   <addaction name="actionFontThrou"/>
   <addaction name="actionFontColor"/>
   <addaction name="separator"/>
   <addaction name="actionToolUndo"/>
   <addaction name="actionToolRedo"/>
  </widget>
  <action name="actionNewFile">
   <property name="text">
    <string>新建项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>打开项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="text">
    <string>保存项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionAboutus">
   <property name="text">
    <string>关于我们</string>
   </property>
  </action>
  <action name="actionReadMe">
   <property name="text">
    <string>项目说明</string>
   </property>
   <property name="toolTip">
    <string>项目说明</string>
   </property>
  </action>
  <action name="actionkaiyuanxieyi">
   <property name="text">
    <string>开源协议</string>
   </property>
  </action>
  <action name="actionSaveAsSVG">
   <property name="text">
    <string>导出为SVG</string>
   </property>
  </action>
  <action name="actionSetBack">
   <property name="text">
    <string>导入背景图片</string>
   </property>
  </action>
  <action name="actionFillColor">
   <property name="text">
    <string>设置填充颜色</string>
   </property>
  </action>
  <action name="actionLineColor">
   <property name="text">
    <string>设置线条颜色</string>
   </property>
  </action>
  <action name="actionSaveTool">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>SaveTool</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontBold">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextBold"/>
   </property>
   <property name="text">
    <string>FontBold</string>
   </property>
   <property name="toolTip">
    <string>设置字体粗体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToolRedo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditRedo"/>
   </property>
   <property name="text">
    <string>ToolRedo</string>
   </property>
   <property name="toolTip">
    <string>重做</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToolUndo">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditUndo"/>
   </property>
   <property name="text">
    <string>ToolUndo</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontItalic">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextItalic"/>
   </property>
   <property name="text">
    <string>FontItalic</string>
   </property>
   <property name="toolTip">
    <string>设置字体斜体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToolNew">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderNew"/>
   </property>
   <property name="text">
    <string>ToolNew</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontUnder">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextUnderline"/>
   </property>
   <property name="text">
    <string>FontUnder</string>
   </property>
   <property name="toolTip">
    <string>设置字体下划线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontThrou">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FormatTextStrikethrough"/>
   </property>
   <property name="text">
    <string>FontThrou</string>
   </property>
   <property name="toolTip">
    <string>设置字体删除线</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveAsPNG">
   <property name="text">
    <string>导出为PNG</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/assets/toolbaricon/FontColor.png</normaloff>:/assets/toolbaricon/FontColor.png</iconset>
   </property>
   <property name="text">
    <string>FontColor</string>
   </property>
   <property name="toolTip">
    <string>设置字体颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToolCut">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCut"/>
   </property>
   <property name="text">
    <string>ToolCut</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToolCopy">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy"/>
   </property>
   <property name="text">
    <string>ToolCopy</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionToolPaste">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste"/>
   </property>
   <property name="text">
    <string>ToolPaste</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFontFamily">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/assets/toolbaricon/FontFamily.png</normaloff>:/assets/toolbaricon/FontFamily.png</iconset>
   </property>
   <property name="text">
    <string>设置字体</string>
   </property>
   <property name="toolTip">
    <string>设置字体</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>view/canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
